docs/HowToAddAnOperation.md

```diff
@@ -31,11 +31,17 @@ Most operations have constraints on their input and parameters. The best way to
 OpsWithVerifier = ['AveragePool', 'Conv', 'InstanceNormalization', 'Mod']
 ```
 
```

Comment:
We did not use OpsWithShapeInference for a long time. Now we generate ShapeInference interface for all Op by default. I am hesitating whether we should bring it back. Previously, we use not implemented shape inference to indicate unsupported ops. You can remove this paragraph.

Reply:
got it, removed it and rephrased it a bit.